### ISSUE TITLE
[Command Button] Color contrast for '+' icon is not good

### ISSUE THREAD

**Issue by ipip2005** [OfficeDev/office-ui-fabric-react#2531]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: Chrome, windows 10 desktop, standard resolution 1920 x 1200.

#### Priorities and help requested (not applicable if asking question):

Requested priority: (Normal) 

Products/sites affected: (web parts)

#### Describe the issue:

The color contrast for the "+" icon specifically is not meeting the requirement of 4.5 : 1

#### Actual behavior:

![image](https://user-images.githubusercontent.com/7711735/29396616-5e310266-834b-11e7-81fa-22d03731be07.png)

#### Expected behavior:

![image](https://user-images.githubusercontent.com/7711735/29396623-64cd8bd0-834b-11e7-8169-9bd441a75300.png)

Note: icon behaves correct in 15px large.
Failed contrast in 16px large.

See more discussions at https://onedrive.visualstudio.com/SOX/_workitems?id=372255&_a=edit&triage=true

**Comment by kristoferbrown** [comment]
@betrue-final-final, themePrimary is fine to use as a color for small text and icons right?

It's looking here like the icon is supposed to be themePrimary but maybe the eyedropper tool in these screenshots is catching one of the anti-aliased pixels. #4a98df is not in our palette anywhere.

@ipip2005, what color does the CSS inspector specify that this element should be?

**Comment by betrue-final-final** [comment]
Yes, themePrimary passes. Note that this issue is in the SharePoint page and authoring version of the command bar because the plus icon is set to 16px. This causes subpixel rendering. There was a PR out to change it to 15px, but not sure what happened to it.

**Comment by kristoferbrown** [comment]
That's what I thought, I'm leaning towards closing this as not an issue with Fabric React but I'd like @ipip2005 to confirm what the css color and font size is set to and if that font-size is coming from fabric or somewhere else.  

We do have a few icons set to 16px, perhaps the right move here is to change them all to 15px since 16 causes subpixeling which just doesn't look as good and causes issues like this.

**Comment by betrue-final-final** [comment]
Actually it's unique to the Add icon because it's one of the few with lines directly center.

**Comment by kristoferbrown** [comment]
OK, in that case I'm closing this as not being an issue with Fabric, just a quirk of living with subpixel rendering of fonts.
